.isort.cfg

```diff
@@ -0,0 +1,16 @@
+[settings]
+line_length=120
+indent=4
+force_grid_wrap=True
+multi_line_output=3
+known_first_party=
+known_third_party=
+sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+import_heading_future=
+import_heading_stdlib=
+import_heading_thirdparty=
+import_heading_firstparty=
+import_heading_localfolder=
```

Comment:
Can we remove those empty assignments?

Reply:
Resolved in 461bee55ae4af25242c74b87dfa5b0360cc188e8